components/brave_sync/brave_sync_service_unittest.cc

```diff
@@ -915,6 +927,11 @@ TEST_F(BraveSyncServiceTest, GetDevicesWithFetchSyncRecords) {
       .Times(1);
   sync_service()->FetchDevices();
 
+  using brave_sync::tools::IsTimeEmpty;
+  EXPECT_TRUE(IsTimeEmpty(sync_service()->this_device_created_time_));
+  // Emulating part of BraveProfileSyncServiceImpl::OnPollSyncCycle()
+  sync_service()->this_device_created_time_ = base::Time::Now();
```

Comment:
we should mock fetch like https://github.com/brave/brave-core/pull/3988/commits/7d58b42c54f8251843b961eda7d62196829ceed6#diff-8b38d039014ebfe5a6c917bf6ee86992R714, otherwise the following `EXPECT_FALSE(IsTimeEmpty(sync_service()->this_device_created_time_));` is ponitless

Reply:
addressed in https://github.com/brave/brave-core/pull/3988/commits/178dbbb5f27e22b93b3e87e745774c6f4f48f605